### ISSUE TITLE
Known building issue on macOS

### ISSUE THREAD

**Issue by pe3zx** [pe3zx/mthc#18]
This issue occurred on macOS 12.2 running engine v20.10.2 and compose v1.29.2.

```
 => ERROR [11/19] RUN bash INSTALL_NODB.sh -A -u                                                                                                       4.4s
------                                                                                                                                                      
 > [11/19] RUN bash INSTALL_NODB.sh -A -u:                                                                                                                  
#16 3.810 sha1 matches                                                                                                                                      
#16 3.821 sha256 matches                                                                                                                                    
#16 3.838 sha384 matches                                                                                                                                    
#16 3.851 sha512 matches                                                                                                                                    
#16 3.852 tput: No value for $TERM and no -T specified
#16 3.854 -
#16 3.911 The following DB Passwords were generated...
#16 3.911 Admin (root) DB Password: 027d6197eff8b59846024cb4fc6559bbfcd30c7ca597d813de94d93be2a5dda1
#16 3.911 User  (misp) DB Password: bd3100a4ffae9e7b757b33fc114503b20f59b7eb5e5765b845d98d510c4a042a
#16 3.912 all
#16 3.912 unattended
#16 3.918     Either your platform is not easily detectable or is not supported by this
#16 3.918     installer script.
#16 3.918     Please visit the following URL for more detailed installation instructions:
#16 3.918     https://misp.github.io/MISP/
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c bash INSTALL_NODB.sh -A -u]: exit code: 1
```

**Comment by pe3zx** [comment]
The issue has been investigated on MISP/misp-docker#141 with pull request submitted. I will update [pe3zx/misp-docker](https://github.com/pe3zx/misp-docker) accordingly after the upstream accept this.

**Comment by fukusuket** [comment]
Thanks for excellent tool :)
[A pull request](https://github.com/MISP/misp-docker/pull/142) for issue [https://github.com/MISP/misp-docker/issues/141](https://github.com/MISP/misp-docker/issues/141) has been merged. If possible, I would appreciate it if you could merge it.

**Comment by pe3zx** [comment]
Change merged at 6391a016eded74ab67810480c7837a5c2812cc9d. [Test build](https://github.com/pe3zx/mthc/actions/runs/3137019974) looks good. Thanks Takahashi san!.
